packages/inter-protocol/test/smartWallet/test-oracle-integration.js

```diff
@@ -177,6 +177,7 @@ test.serial('invitations', async t => {
    * @returns {Promise<[{description: string, instance: Instance}]>}
    */
   const getInvitationFor = async (desc, len, balances) => {
+    await eventLoopIteration();
```

Comment:
I often feel obliged to say why I'm adding `await eventLoopIteration()`. I wonder about an alias such as `await vstorageWritesVisible()`.